### DISCUSSION TITLE
Display names of polygons

### DISCUSSION THREAD

**Discussion by QuestionsForThose** [osmandapp/OsmAnd#10965] [Q&A]
Often area polygons tags are sitting (correctly) on the area.
In these instances OSM cannot display the names or the use tag.

Is there a setting I can change to fix this?

**Comment by tommycrock** [comment]
Maybe I misunderstand. I can see the names of buildings and landuse areas on the map with all extra details switched off, if I'm zoomed in enough, e.g. 
![Screenshot_20210221-092606_OsmAnd+](https://user-images.githubusercontent.com/5749884/108621075-61028100-7428-11eb-9fbd-6f2a7bfa12bf.jpg)
![Screenshot_20210221-093829_OsmAnd+](https://user-images.githubusercontent.com/5749884/108621122-9d35e180-7428-11eb-8861-b576182f674b.jpg)
